### PR TITLE
Docs update 07 05

### PR DESCRIPTION
**Pull request by imponenm**
Minor updates to README files

### CODE REVIEW

File: verifier-agent/node/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # Verifier Agent Example
 
-This folder contains an example Node.js application that demonstrates integrating with GlobaliD as a [Verifier](https://www.w3.org/TR/vc-data-model/#dfn-verifier). The example is built with [NestJS](https://nestjs.com/) and uses [GlobaliD's Verifier Toolkit](https://npmjs.com/package/@globalid/verifier-toolkit).
+This folder contains an example Node.js application that demonstrates integrating with GlobaliD as a [Verifier](https://www.w3.org/TR/vc-data-model/#dfn-verifier). In this case, the application validates the end-user is 18 or older.
```
Comment (dillonredding) [inline verifier-agent/node/README.md:3]:
```suggestion
This folder contains an example Node.js application that demonstrates integrating with GlobaliD as a [Verifier](https://www.w3.org/TR/vc-data-model/#dfn-verifier). In this case, the application validates the [holder](https://www.w3.org/TR/vc-data-model/#dfn-holders) is 18 or older using either their driver's license or passport credential.
```

File: README.md [hunk 2]
```diff
@@ -1,3 +1,7 @@
 # GlobaliD Developer Tooling Examples
 
-This repository contains examples utilizing GlobaliD's developer tooling.
+This repository contains Node.js examples that utilize GlobaliD's developer tooling. It is a monorepo which contains separate sample projects.
+
+## Sample apps
+- **[globalid-connect](https://github.com/globalid/dev-tooling-examples/tree/main/globalid-connect)**: Demonstrates login flow using GlobaliD
+- **[verifier-agent](https://github.com/globalid/dev-tooling-examples/tree/main/verifier-agent)**: Demonstrates verification flow for validating a user is 18+
```
Comment (dillonredding) [inline README.md:7]:
```suggestion
- [GlobaliD Connect](./globalid-connect): Demonstrates how to log in using GlobaliD
- [Verifier Agent](./verifier-agent): Demonstrates Verifier flow for validating a user is 18+
```

**Review by dillonredding — DISMISSED** [review]
Some minor cleanup/clarification

**Comment by imponenm** [comment]
@dillonredding Thanks for those suggestions, committed them

**Pull request merged**